content/services/opence.rst

```diff
@@ -0,0 +1,186 @@
+Open-CE
+=======
+:slug: opence
+:author: Ken Lett
+:menu: POWERLinux/OpenPOWER Development Hosting, Open-CE, 3
+
+The Open Source Lab (OSUOSL) and Center for Genome Research and Biocomputing (CGRB) partner with IBM and OpenPOWER in order to provide a download resources around Open-CE. Open-CE is a community driven software distribution for machine learning that runs on standard Linux platforms with NVIDIA GPU technologies.
+- `Current release`_
+
+- `Previous releases`_
+
+.. _Current release:
+.. _Release 1.0.0:
+
+Open-CE Release 1.0.0
+---------------------
+
+*Release date: 11/10/2020*
+
+**What's new**
+
+Open-CE 1.0 is the `current release`_ of Open-CE and includes the following features:
+
+- conda packages are now available on ppc64le.
+- conda packages are now available on x86.
+- TensorFlow 2.3.1
+- PyTorch 1.6.0
+- Open-CE is distributed as prebuilt containers, or on demand through the Conda provisioning process.
+
+  - All of the Conda packages are available in a `Open-CE Conda channel`_
+  - Current Conda packages are available in a `Current Open-CE Conda channel`_
+  - There is no install package to download, instead connect to the Conda channel and install your packages from there
+  - Package dependencies are automatically resolved
+  - Delivery of packages is open and continuous
```

Comment:
```suggestion
  - Delivery of packages are open and continuous
```

Reply:
Hmm, I think delivery is the subject, not packages, so 'is' is correct - but it it doesn't read right, will change